api/LargestContentfulPaint.json

```
@@ -187,6 +187,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cross-origin": {
+          "__compat": {
+            "description": "Cross-origin",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/LargestContentfulPaint/renderTime#cross-origin_image_render_time",
+            "tags": [
+              "web-features:largest-contentful-paint"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "133"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
```

Since only Chrome supports this, we should mark it as experimental (I'm surprised the linter didn't complain):

```suggestion
              "experimental": true,
```